### ISSUE TITLE
TypeScript error when assigning to an entity field with schema type "Array"

### ISSUE THREAD

**Issue by chowlett** [sensedeep/dynamodb-onetable#34]
The specific error when assigning, for example, an array of strings to such a field is "error TS2322: Type 'string' is not assignable to type 'undefined'." Reproducible on 1.3.6.

You may want to consider the following possible fix. It worked for me in 1.3.6.

```
--- a/src/Model.d.ts
+++ b/src/Model.d.ts
@@ -91,7 +91,7 @@ type EntityField<T extends OneTypedField> =
     : T['type'] extends BooleanConstructor ? boolean
     : T['type'] extends ObjectConstructor ? object
     : T['type'] extends DateConstructor ? Date
-    : T['type'] extends ArrayConstructor ? []
+    : T['type'] extends ArrayConstructor ? any[]
     : never;
```

**Comment by mobsense** [comment]
Thank you.

**Comment by mobsense** [comment]
Fixed in 1.3.7
